### ISSUE TITLE
BigNumber Error: new BigNumber() not a base 16 number

### ISSUE THREAD

**Issue by PeterYinusa** [MetaMask/metamask-extension#11940]
**Steps to reproduce**

1. Login to the extension
2. Switch to a custom network e.g. `Binance Smart Chain`
3. Click the Send button on the Home page
4. Select a recipient on the Add Recipient page
5. Prepend a `-` to the Gas Price on the Send page

**Expected behavior**

- Error class on the `Gas Price (GWEI)` field
- Next button is disabled

**Error Messages**

- BigNumber Error: new BigNumber() not a base 16 number: 0x-6fc23ac00

```
BigNumber Error: new BigNumber() not a base 16 number: 0x-6fc23ac00
  at ee(metamask/common-0.js:18:72957)
  at S(metamask/common-0.js:18:78437)
  at new z(metamask/common-0.js:18:70451)
  at T(metamask/common-1.js:1:190480)
  at P(metamask/common-1.js:1:191128)
  at n.calcGasTotal(metamask/common-1.js:1:288315)
  at Object.calculateGasTotal(metamask/common-3.js:12:22758)
  at updateGasFees(metamask/common-3.js:12:23626)
  at ? (metamask/common-4.js:10:302433)
  at produce(metamask/common-3.js:19:122502)
  at ? (metamask/common-4.js:10:302410)
  at Array.reduce(<anonymous>)
  at ? (metamask/common-4.js:10:302279)
  at ? (metamask/common-5.js:27:14440)
  at y(metamask/common-5.js:27:15359)
  at ? (metamask/common-5.js:27:16149)
  at dispatch(metamask/common-5.js:27:13586)
  at ? (metamask/common-3.js:12:12533)
  at ? (metamask/common-5.js:27:16140)
  at updateGasPrice(metamask/ui-5.js:10:153760)
  at Object.updateCustomGasPrice(metamask/ui-5.js:10:30783)
  at d.<anonymous>(metamask/ui-5.js:10:328735)
  at m(metamask/common-0.js:1:72601)
  at v(metamask/common-0.js:1:72852)
  at _(metamask/common-0.js:1:72754)
  at a(metamask/sentry-install.js:1:10465)
```

**Comment by seaona** [comment]
Encountered a similar issue on Ethereum Mainnet, by including `-` into edit gas fees. This throws Metamask Encountered an Error page:

https://user-images.githubusercontent.com/54408225/181453472-0e2f6547-a500-46cd-a397-23b465d5bc79.mp4

**Comment by PeterYinusa** [comment]
Resolved in [v10.26](https://github.com/MetaMask/metamask-extension/pull/17698)
